paper/src/main/resources/config.yml

```diff
@@ -74,6 +74,29 @@ reinforcements:
     acid_time: 1s
     name: Bedrock
     hit_points: 420420420
+
+acids:
+  gold:
+    material: GOLD_BLOCK
+    multiplier: 1
+    faces:
+        - UP
+  diamond:
+    material: DIAMOND_BLOCK
+    multiplier: 2
+    faces:
+        - UP
+        - DOWN
+  netherite:
+    material: NETHERITE_BLOCK
+    multiplier: 4
```

Comment:
Agree with gjum that this is best renamed to `maturation_time_multiplier`

Reply:
Updated in c0fa6c1a841854ad34b6ddb5edf321e084efbe94 (Thank you @Gjum)